pkg/storage/db_test.go

```diff
@@ -732,6 +732,303 @@ func TestGetHostnamesAtTimeMultiRowsSchema2(t *testing.T) {
 	}
 }
 
+func TestGetARNIDAtTime(t *testing.T) {
+	mockdb, mock, err := sqlmock.New()
+	if err != nil {
+		t.Fatalf("an error '%s' was not expected when opening a stub database connection", err)
+	}
+	defer mockdb.Close()
+
+	thedb := DB{
+		sqldb: mockdb,
+	}
+
+	withSchemaVersion(4, mock)
+	at, _ := time.Parse(time.RFC3339, "2019-04-09T08:55:35+00:00")
+	const arnID = "arnid"
+
+	rows := sqlmock.NewRows([]string{"aws_private_ip_assignment_private_ip",
```

Comment:
info: I could be beneficial to try to reduce the amount of duplication here. These test rows are created 3x times and then populated with different testing data depending on the scenario, sometimes shared data and sometimes different. You might consider using a technique like table driven tests https://github.com/golang/go/wiki/TableDrivenTests to reduce duplication in the tests.

Reply:
I am open to this being improved as a follow up ticket/PR should you feel the scope/time on this ticket is already larger than you intended, also if for some reason table based tests would not work here then the duplication might be ok.

Reply:
As all inputs for the test cases are the same, I think table driven tests will not work in this case. Instead, I can remove duplicate by creating a global variable, if you prefer.

Reply:
This will likely we fixed in a separate change as table based tests had issues in this spot.